src/appleseed/renderer/modeling/input/texturesource.cpp

```diff
@@ -36,6 +36,7 @@
 #include "renderer/modeling/texture/texture.h"
 
 // appleseed.foundation headers.
+#include "foundation/image/color.h"
```

Comment:
In a future PR: you can remove this as it's already included by `texturesource.h`.

Reply:
done